Android Studio cannot find symbol class RNCWebViewPackage

I tried @FaiChou's fix to comment out RequireAPI, but am still receiving errors building with this package not being found by Android Studio even though it appears to be properly linked and exists within node_modules.

settings.gradle contains:
`include ':react-native-webview'
project(':react-native-webview').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-webview/android')`

MainApplication.java contains:
`import com.reactnativecommunity.webview.RNCWebViewPackage;`

`npm 6.4.1
react-native 0.57.7
node 10.13.0`

am I missing something?

Facing the face issue

> Task :react-native-webview:compileDebugJavaWithJavac FAILED
node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModule.java:11: error: cannot find symbol
import android.support.annotation.RequiresApi;
                                 ^
  symbol:   class RequiresApi
  location: package android.support.annotation
node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModule.java:153: error: cannot find symbol
  @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
   ^
  symbol:   class RequiresApi
  location: class RNCWebViewModule
Note: node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -17,6 +17,7 @@ apply plugin: 'kotlin-android'
 def DEFAULT_TARGET_SDK_VERSION = 27
 def DEFAULT_COMPILE_SDK_VERSION = 27
 def DEFAULT_BUILD_TOOLS_VERSION = "28.0.3"
+def DEFAULT_SUPPORT_LIB_VERSION = "28.0.0"
 
 def getExtOrDefault(name, defaultValue) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : defaultValue
@@ -48,6 +49,7 @@ android {
 repositories {
   mavenCentral()
   jcenter()
+  google()
 
   def found = false
   def defaultDir = null
@@ -115,8 +117,11 @@ repositories {
   }
 }
 
+def support_version = getExtOrDefault('supportLibVersion', DEFAULT_SUPPORT_LIB_VERSION)
+
 dependencies {
   //noinspection GradleDynamicVersion
   api 'com.facebook.react:react-native:+'
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+  implementation "com.android.support:appcompat-v7:$support_version"
 }

